<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plugin::ip::SettingsWidget</class>
 <widget class="QWidget" name="plugin::ip::SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QCheckBox" name="show_internal_addr_cbox">
     <property name="text">
      <string>Display internal address(es)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTreeView" name="interfaces_list">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="show_external_addr_cbox">
     <property name="text">
      <string>Display external address</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="show_external_v4_cbox">
     <property name="text">
      <string notr="true">IPv4</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="show_external_v6_cbox">
     <property name="text">
      <string notr="true">IPv6</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>show_internal_addr_cbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>interfaces_list</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_external_addr_cbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>show_external_v4_cbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_external_addr_cbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>show_external_v6_cbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
